packages/components/src/Breakout/Breakout.tsx

```diff
@@ -24,19 +25,22 @@ const useAlignment = (direction?: string) => {
 export const Breakout = ({
   horizontalAlign,
   verticalAlign,
+  height,
   children,
   ...props
 }: BreakoutProps) => {
-  const alignItems = useAlignment(horizontalAlign);
-  const justifyContent = useAlignment(verticalAlign);
+  const alignItems = useAlignment(verticalAlign);
+  const justifyContent = useAlignment(horizontalAlign);
+
   return (
     <Box
       alignItems={alignItems}
       justifyContent={justifyContent}
       width="100%"
+      height={height}
       display={verticalAlign || horizontalAlign ? 'flex' : 'block'}
-      css={{
-        gridColumn: '1 / -1',
+      __baseCSS={{
+        gridColumn: '1 / -1 !important',
```

Comment:
Do we still need this?

Reply:
i think yes because it takes the gridColumn from the parentelement if it isn't set